apps/examples/wifi_manager_sample/wm_test_stress_test1.c

```diff
@@ -84,6 +86,22 @@ void wm_sta_disconnected(wifi_manager_disconnect_e disconn)
 	WM_TEST_SIGNAL;
 }
 
+void wm_softap_sta_joined(void)
+{
+	printf(" T%d --> %s\n", getpid(), __FUNCTION__);
+}
+
+void wm_softap_sta_left(void)
+{
+	printf(" T%d --> %s\n", getpid(), __FUNCTION__);
+}
+
+void wm_scan_ap_done(wifi_manager_scan_info_s **info, wifi_manager_scan_result_e res)
```

Comment:
Why don't you use 'info' here in case the scan is successful?

Reply:
it's a test program so I think it doesn't need to print scan result which take time. Instead of it I add logic whether a pointer of info is valid or not.